inc/saitunnel.h

```diff
@@ -463,7 +462,6 @@ typedef enum _sai_tunnel_attr_t
      * @type sai_object_id_t
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY
      * @objects SAI_OBJECT_TYPE_ROUTER_INTERFACE
-     * @condition SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_IPINIP or SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_IPINIP_GRE
```

Comment:
For VXLAN, do we need overlay interface?

Reply:
@MELLANOX-zhichenr I also thought that we do not need that. But configuration of SAI_TUNNEL_OVERLAY_INTERFACE attribute for VXLAN tunnel is given in [SAI-Proposal-tunnel_v6](https://github.com/opencomputeproject/SAI/blob/master/doc/tunnel/SAI-Proposal-tunnel_v6.docx) (page 23).

Reply:
There is an updated proposal for VXLAN tunnel with .1D bridge. In this case, I guess we need to use tunnel map <vni, bridge if> instead of overlay interface.

https://github.com/opencomputeproject/SAI/blob/master/doc/bridge/SAI_Proposal_Bridge_port_v0.9.docx

I guess the original tunnel proposal needs to be updated.

Thus I think we need to keep the comment for overlay interface. The other changes look fine to me.

Reply:
@MELLANOX-zhichenr, ok. I will revert condition section for SAI_TUNNEL_OVERLAY_INTERFACE attribute.

Reply:
Thanks! The new change looks good to me.

Reply:
@MELLANOX-zhichenr  @IGordynskyi Wouldn't overlay interface needed for VXLAN for L3 gateway or RIOT feature ? Say after  VXLAN termination we want to route the inner packet then the overlay interface would be needed i guess ?

Reply:
@karthik-krishnamurthy / @MELLANOX-zhichenr , yes it looks like we need SAI_TUNNEL_OVERLAY_INTERFACE attribute for L3 gate way functionality. Please see VXLAN decap pipeline object model: 
![vxlan_decap](https://user-images.githubusercontent.com/22078793/34108073-4e873dae-e407-11e7-95b3-a37ed9bc5757.png)

Reply:
@MELLANOX-zhichenr Do you have any concerns on this?

Reply:
I am OK with the change since there is a use case for it.